src/tildePrefix.ml

```diff
@@ -68,5 +76,7 @@ let rec make_tilde_prefix_explicit rhs_assignment = function
   | cst ->
     [cst]
 
+(** Crawls through the [word_component]s of a [word] and process them. See
```

Comment:
`process` -> `processes`?

Or: use the imperative form in such comment?

Reply:
I think we've used third person in most of Morbig. In this particular case, the comment has since then changed anyways.